### PR TITLE
Close #112 (separate Thing and sdfThing)

### LINKED ISSUE #112: (editorial): Thingness (CH-22), sdfThing vs. Thing (CH-23); also CH-7, CH-8, CH-9, CH-11
Also, clearly separate Thing (generic IoT term) from sdfThing

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -1449,7 +1447,10 @@ consumption there is no conflict with the intended goal.
 
 ## sdfThing
 
-An sdfThing is a set of declarations and qualities that may be part of a more complex model. For example, the object declarations that make up the definition of a single socket of an outlet strip could be encapsulated in an sdfThing, and the socket-thing itself could be used in a declaration in the sdfThing definition for the outlet strip
+An sdfThing is a set of declarations and qualities that may be part of
+a more complex model. For example, the object declarations that make
```
Comment (JKRhb) [inline sdf.md:1451]: Should "object" be changed to "sdfObject" or "Object" here?

Comment (cabo) [reply sdf.md]: Good point.  Fixed in commit below.

File: sdf.md [hunk 2]
```diff
@@ -139,8 +139,9 @@ entity:
 
 [^intro-]:
     The Semantic Definition Format (SDF) is a format for domain experts to
-    use in the creation and maintenance of data and interaction models in
-    the Internet of Things. An SDF specification describes definitions of
+    use in the creation and maintenance of data and interaction models
+    that describe Things, i.e., physical objects that are available
```
Comment (akeranen) [inline sdf.md:143]:
```suggestion
    that describe Things, i.e., physical objects that are available for interaction
```

File: sdf.md [hunk 3]
```diff
@@ -168,13 +169,10 @@ entity:
...
 
 Thing:
-: A physical item that is also made available in the Internet of
-  Things.  The term is used here for Things that are notable for their
-  interaction with the physical world beyond interaction with humans;
-  a temperature sensor or a light might be a Thing, but a router that
-  employs both temperature sensors and indicator lights might exhibit
-  less Thingness, as the effects of its functioning are mostly on the
-  digital side.
+: A physical item that is also available over a network.
```
Comment (akeranen) [inline sdf.md:172]:
```suggestion
: A physical item that is also available for interaction over a network.
```

Although maybe we don't need this term now? Or should we say that "Thing" is here simply a shorthand for SDF Thing?

**Review by akeranen — APPROVED** [review]

File: sdf.md (hunk 3 above)
Comment (cabo) [reply sdf.md:172]: Object and sdfObject are the same, Thing and sdfThing are completely different.

**Pull request merged**